server/mailet/mailets/src/main/java/org/apache/james/transport/util/SizeUtils.java

```diff
@@ -23,12 +23,13 @@
 
 public class SizeUtils {
 
+    /**
+     * Return a human-readable version of the file size, where the input represents a specific number of bytes.
+     *
+     * @param size >= 0
```

Comment:
I think that supporting negative numbers is better

Reply:
Ok ok I revert the commit..